networking/multiple_networks/secondary_networks/configuring-multi-network-policy.adoc

```diff
@@ -6,6 +6,9 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+Multi-network policy is used by administrators to manage secondary networks created using the network attachment definition (NAD) resource. Administrators use the `MultiNetworkPolicy` API to manage traffic for pods that are attached to multiple networks. For example, you can separate the control plane and data plane traffic.
```

Comment:
This is not explaining core concept of multi net policies. It does not manage a interface

Reply:
This is existing docs can we file a bug for this?

Reply:
@tssurya

Reply:
@JoeAldinger Any insights on this? Should I revert it back to its original state prior to the reorg? Like [this](https://docs.openshift.com/container-platform/4.17/networking/multiple_networks/configuring-multi-network-policy.html#:~:text=As%20a%20cluster,are%20not%20supported.)?